search/indexing_api.py

```diff
@@ -304,6 +322,16 @@ def clear_index():
     create_mappings()
 
 
+def delete_index():
```

Comment:
can you remove the repeated code in `clear_index` and call this function instead?